db.js

```diff
@@ -1,30 +1,30 @@
 import { createRequire } from "module";
 import { CoursesSchema } from "./src/models/Courses.js";
 import { AdminsSchema } from "./src/models/admin.js";
-import dotenv from "dotenv"
+
+import configs from "./src/config/config.js";
+import { SeminarsSchema } from "./src/models/Seminars.js";
+import { WorkshopsSchema } from "./src/models/Workshops.js";
+import { DiplomasSchema } from "./src/models/Diplomas.js";
 const require = createRequire(import.meta.url);
 const typeorm = require("typeorm");
-dotenv.config()
+
 export const dataSource = new typeorm.DataSource({
   type: "mysql",
-  host: process.env.DB_HOST,
-  port: process.env.DB_PORT,
-  username: process.env.DB_USERNAME,
-  password: process.env.DB_PASSWORD,
-  database: process.env.DB_NAME,
-//   synchronize: true,
-  entities: [CoursesSchema, AdminsSchema],
+  host: configs.database.host,
+  port:configs.database.port,
+  username: configs.database.username,
+  password: configs.database.password,
+  database: configs.database.database,
+  //   synchronize: true,
+  entities: [CoursesSchema, AdminsSchema, SeminarsSchema, WorkshopsSchema, DiplomasSchema],
```

Comment:
Issue(blocking): I don't understand why you use one table by entity type with the same fields. I prefer once table with a field "type" for easy maintenance. The current implementation will have a lot of issues for future features.

Reply:
Done